.github/workflows/schedule.yml

```diff
@@ -0,0 +1,28 @@
+name: Runs e2e every day at 2pm against RT Prod
+on:
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
as disc, let's run this at 9am to start with, and I think aim it at Dev system until we are happy with the tests and the timing